package.json

```diff
@@ -21,16 +21,20 @@
     "url": "https://github.com/DefinitelyTyped/focus-dt/issues"
   },
   "devDependencies": {
+    "@octokit/rest": "^21.0.0",
     "@types/color-convert": "^2.0.0",
     "@types/node": "^16.11.47",
     "@types/winreg": "^1.2.30",
     "@types/yargs": "^13.0.0",
-    "typescript": "^4.7.4"
+    "octokit": "^4.0.2",
```

Comment:
Was this intended to be a prod dep?

Reply:
Fixed, thanks. I was too aggressive with `-D`